ci/deps/travis-36-cov.yaml

```diff
@@ -33,8 +33,7 @@ dependencies:
   # https://github.com/pydata/pandas-gbq/issues/271
   - google-cloud-bigquery<=1.11
   - psycopg2
-  # pyarrow segfaults on load: https://github.com/pandas-dev/pandas/issues/26716
-  # - pyarrow=0.9.0
+  - pyarrow>=0.12.0
```

Comment:
if the new minimum version is 0.12.0 and not 0.9.0 we should update the documentation

Reply:
That was already done in a previous PR by Brock (https://dev.pandas.io/docs/getting_started/install.html#optional-dependencies)